bitnami/prometheus/templates/alertmanager/pdb.yaml

```diff
@@ -0,0 +1,28 @@
+{{- $replicaCount := int .Values.alertmanager.replicaCount }}
+{{- if and .Values.alertmanager.enabled .Values.alertmanager.pdb.create (or (gt $replicaCount 1) .Values.alertmanager.autoscaling.enabled) }}
```

Comment:
There is no HPA object for Alertmanager, so no need to check if alertmanager.autoscaling.enabled

Reply:
Good catch!